.rubocop_todo.yml

```diff
@@ -68,27 +68,27 @@ RSpec/DescribeClass:
     - '**/spec/system/**/*'
     - '**/spec/views/**/*'
     - 'spec/unit/forge_helper_spec.rb'
-    - 'spec/unit/metadata_checker.rb'
+    - 'spec/unit/metadata_checker_spec.rb'
     - 'spec/unit/runner_spec.rb'
 
 # Offense count: 5
 # Configuration parameters: CountAsOne.
 RSpec/ExampleLength:
   Max: 7
 
+# Offense count: 1
+# Configuration parameters: Include, CustomTransform, IgnoreMethods, SpecSuffixOnly.
+# Include: **/*_spec*rb*, **/spec/**/*
+RSpec/FilePath:
+  Exclude:
+    - 'spec/unit/metadata_checker.rb'
```

Comment:
this can probably be removed now, you can just regenerate the todo :)

Reply:
Yes!  Nice eyes.

Reply:
Removed.